test/js/components/projects/detail.test.jsx

```diff
@@ -1239,16 +1252,20 @@ describe('<ProjectDetail />', () => {
 
   describe('fetching more tasks', () => {
     test('fetches next page of tasks', async () => {
-      const { findByText, getByText, getAllByText } = setup({
+      const { findByText, getAllByText } = setup({
         initialState: {
           ...defaultState,
           tasks: {
             p1: {
               tasks,
-              next: {},
+              next: {
+                all: 'next-task-url',
+              },
```

Comment:
Because this was missing, there was no "next page" of tasks, so there wasn't a Task "Load More" btn. So the click below was actually clicking the *Epic* "Load More" btn.